charts/fluid/fluid/templates/role/csi/rbac.yaml

```diff
@@ -0,0 +1,117 @@
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: fluid-csi-admin
+  namespace: fluid-system
+---
+kind: ClusterRole
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: fluid-csi-plugin
+rules:
+  - apiGroups:
+      - data.fluid.io
+    resources:
+      - alluxiodataloads
+      - alluxioruntimes
+      - datasets
+      - alluxiodataloads/status
+      - alluxioruntimes/status
+      - datasets/status
+    verbs:
+      - '*'
+  - apiGroups:
+      - apps
+    resources:
+      - daemonsets
+      - statefulsets
+      - daemonsets/status
+      - statefulsets/status
+    verbs:
+      - '*'
+  - apiGroups: [""]
```

Comment:
Let's review the RBAC authorization. Why each item is required?

Reply:
Fixed. The RBAC has been minimized.